pages/index.tsx

```diff
@@ -99,9 +118,41 @@ const HeaderSection = (props: HomePageHeader) => {
           </Typography>
         </Box>
       </Box>
-      <Typography variant="body_normal" color="#fff" sx={{ maxWidth: 950, mx: 'auto' }}>
-        {props.subtitle}
```

Comment:
i pasted the URL of the preview site in slack and when it unfurled it, it looks to still be displaying the old copy. Is there a way to update that copy also to this new messaging?
https://www-crossplane-git-fork-jeanduplessis-vee-two-preview-upboundio.vercel.app/

Reply:
Yes, I'll update the page meta data as well

Reply:
i wonder if this is just a preview site thing though, where the URL unfurling gets its value from main or the live site and not the branch or preview site 🤔

Reply:
Its because of the formatting yes. The way it was set up couldn't handle the unordered list. The easy solution, without changing how things currently work, was to replace subtitle with custom code.